src/matlab/flowdevicemethods.cpp

```diff
@@ -1,5 +1,5 @@
 // This file is part of Cantera. See License.txt in the top-level directory or
-// at http://www.cantera.org/license.txt for license and copyright information.
+// at http://cantera.org/license.txt for license and copyright information.
```

Comment:
I think always using `https` is the main change that Bryan wants to have in these; omitting the `www` is secondary.

Reply:
corrected.